c2cgeoportal/models.py

```diff
@@ -1021,10 +1022,12 @@ class Dimension(Base):
             cascade="save-update,merge,delete,delete-orphan",
         ),
     )
+    field = Column(Unicode, label=_(u"Field"))
```

Comment:
Why not put it just around the description?

Reply:
Because I like field to appear after layer in admin UI

Reply:
The field order can be change like this:
https://github.com/arnaud-morvan/c2cgeoportal/blob/f4f07948167539be05f9291838a13ffbfbeee05e/c2cgeoportal/forms.py#L704-L709

Reply:
And it's another thing in 2.3

Reply:
Yes but as I know you didn't move the fields in the model to change the order, or yes?

Reply:
As I remember, we did not change any order, we show fields in sqlalchemy model order.
Except for some relations, we create schema manually and insert them imperatively at the right place.
Concerning dimensions, as they are embedded in layer form, layer field should be excluded.
So you're right, we could put field before layer.

Reply:
cool :-)